src/index.js

```diff
@@ -0,0 +1,26 @@
+const url = "https://api.github.com/search/users?q=";
+const userMode = "%20in:login";
+let item = document.querySelector(".items");
+const button = document.querySelector(".search_button");
+button.addEventListener("click", displayUser)
+
+async function getResource() {
+  let nameMode = document.querySelector(".search_input").value;
+  if (nameMode) {
+    const res = await fetch(url + nameMode + userMode);
+    // if (!res.ok) {
+    //     throw new Error(`Could not fetch ${url}` +
+    //         `, rceived ${res.status}`)
+    // }
+    return await res.json();
+  };
+};
+
+async function displayUser() {
+  const res = await getResource();
```

Comment:
not `res` :) you received `users`! so help yourself and make your code more self-descriptive ;)